core/tests/protractor_utils/forms.js

```diff
@@ -51,7 +52,7 @@ var GraphEditor = function(graphInputContainer) {
   var createVertex = async function(xOffset, yOffset) {
     var addNodeButton = graphInputContainer.element(
       by.css('.protractor-test-Add-Node-button'));
-    await addNodeButton.click();
+    await action.click('Test Add Node Button', addNodeButton);
     // Offsetting from the graph container.
     await browser.actions().mouseMove(
       graphInputContainer, {x: xOffset, y: yOffset}).perform();
```

Comment:
Can we make sure that the graphInputContainer is visible first?

Reply:
Done!